src/Common/src/Interop/User32/Interop.CS.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        /// <summary>
+        ///  Class styles
+        /// </summary>
+        public enum CS : uint
```

Comment:
In the spirit and in line with other refactors:
```suggestion
        public enum ClassStyle : uint
```

Looking at the use, it probably should be decorated as `Flags`, and then name `ClassStyles`

Also there are more members, perhaps it is worth adding them all https://github.com/tpn/winsdk-10/blob/master/Include/10.0.14393.0/um/WinUser.h#L2856-L2874

Reply:
As per https://github.com/dotnet/winforms/pull/1742#discussion_r321841577 I don't think that is the right choice.

For diagnostic purposes I think having the additional defines are useful, but they do bloat the size of the assembly, so we should be thinking about the value in each case and not doing it blindly. One option is to comment out additional defines when we only use a few.

Reply:
I've changed a few of the existing ones to this pattern. Here is an example of where it really starts making a difference:

https://github.com/dotnet/winforms/pull/1780/commits/0abd60a6aa3f062171a96158ca0dd5e66ccb55df#diff-c6a8af157a9eb8becc613c8d53ce2c1fL622

Reply:
I can't say I find it particularly exciting, but at least it is consistent. 👍